pkg/backend/cloudwatch/client.go

```diff
@@ -0,0 +1,151 @@
+package cloudwatch
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/egnyte/ax/pkg/backend/common"
+)
+
+type CloudwatchClient struct {
+	client    *cloudwatchlogs.CloudWatchLogs
```

Comment:
Personally I'd call this `logs`:

- Consistency with the type
- Avoid `client.client`

Reply:
Makes sense

Reply:
Done